### PR TITLE
Talons

### PR DESCRIPTION
**Pull request by rfavour**

### CODE REVIEW

**Comment by floogulinc** [comment]
So by the way, a pull request will always track the same branch. You can make a PR and then push more to the branch and it will be updated in the PR. You don't need to close and create a new PR.

File: src/main/java/com/frc2879/newcomen/subsystems/Drivetrain.java [hunk 1]
```diff
@@ -1,31 +1,52 @@
...
+        initTalonConfig();
 
     }
 
-
+    public void initTalonConfig() {
+	    TALONS_LF = 1;
+	    TALONS_LB = 2;
+	    TALONS_RF = 3;
+	    TALONS_RB = 4;
```
Comment (floogulinc) [inline src/main/java/com/frc2879/newcomen/subsystems/Drivetrain.java:32]: So these are temporary until we implement a config system.

**Review by floogulinc — APPROVED** [review]
Looks good

File: src/main/java/com/frc2879/newcomen/subsystems/Drivetrain.java (hunk 1 above)
Comment (rfavour) [reply src/main/java/com/frc2879/newcomen/subsystems/Drivetrain.java:32]: yes

**Pull request merged**